apps/explorer/lib/contracts.ts

```diff
@@ -0,0 +1,21 @@
+import { ExplorerFileContract } from '@siafoundation/explored-types'
+
+export enum CONTRACT_STATUS {
```

Comment:
I would rather not introduce enums into our codebase (https://www.youtube.com/watch?v=jjMbPt_H3RQ). Lets either do a `const` object or a union type.